templates/serverpod_templates/github/workflows/deployment-docker.yml

```diff
@@ -0,0 +1,121 @@
+# This is a workflow to deploy your Serverpod to a single machine using docker-compose. This is useful for testing and small deployments. For larger deployments, you should use the deployment-aws.yml or deployment-gcp.yml workflows.
+# To reduce the workload on the machine we do not use redis in this deployment. If you want to use redis, you need to add it to the docker-compose file and the serverpod configuration.
+# You need to setup the correct hostnames in the docker-compose-production file AND the serverpod configuration file.
+# Before deploying this workflow, you need to set up the following secrets in your repository settings:
+# GITHUB_TOKEN: A GitHub token with the repo scope
```

Comment:
GitHub secret names must not start with "GITHUB_"

Reply:
EDIT - apparently every workflow automatically has a GITHUB_TOKEN set. this can be removed here